neoload/README.md

```diff
@@ -0,0 +1,48 @@
+# NeoLoad integration
+
+## Overview
+
+[Tricentis NeoLoad][1] simplifies and scales performance testing for everything, from APIs and microservices, to end-to-end application testing through innovative protocol and browser-based capabilities.
+
+With the NeoLoad integration, you can track performance metrics of NeoLoad tests to:
+
+- Correlate application performance with load testing metrics.
+- Analyze and visualize NeoLoad metrics using the NeoLoad Datadog Dashboard or Metrics Explorer.
+
+## Setup
+
+### Configuration
+
+The Datadog agent is not needed to use the integration.
+For the detailed instructions on NeoLoad configuration, follow the [NeoLoad documentation][2].
+Additionally, the first time Datadog detects the NeoLoad.Controller.User.Load metric, the NeoLoad integration tile is installed automatically, and the default NeoLoad dashboard is added to your dashboard list.
```

Comment:
```suggestion
For detailed instructions on NeoLoad configuration, follow the [NeoLoad documentation][2].
When Datadog receives `NeoLoad.*` metrics, the NeoLoad integration tile is installed automatically, and the default NeoLoad dashboard is added to your dashboard list.
```

Reply:
Could you provide more detail about how the user sends `Neoload.*` metrics to Datadog? The provided link leads to multiple options, so any additional clarification here will be helpful for our users.

Reply:
You're right, there is a few missing informations, I will add this.
To send events and to send metrics with NeoLoad, the user must use the version 9.1 of NeoLoad that should be published shortly (it is ready waiting for approvals).
The link to the doc will be updated when publishing with more details on the integration.
It will contain a screenshot (the user can choose what to send. Everything is checked by default).

Reply:
![image](https://user-images.githubusercontent.com/57721694/199698551-cd7a62c3-61cc-4b91-b1ee-96f5c482317e.png)